packages/block-editor/src/components/font-appearance-control/style.scss

```diff
@@ -1,8 +1,6 @@
 .components-font-appearance-control {
-	ul {
-		li {
-			color: $gray-900;
-			text-transform: capitalize;
-		}
+	[role="option"] {
```

Comment:
Switching from `ul > li` to `[role="option"]` was necessary because the new version of `CustomSelectControl` doesn't use `ul` and `li`s — and in general it makes these styles more resilient to future changes.